Index: src/main/java/com/knight/stream/Test.java
===================================================================
diff --git a/src/main/java/com/knight/stream/Test.java b/src/main/java/com/knight/stream/Test.java
deleted file mode 100644
--- a/src/main/java/com/knight/stream/Test.java	(revision b4d89eae9b533a677cf08c9a8404e8d98f0cf6af)
+++ /dev/null	(revision b4d89eae9b533a677cf08c9a8404e8d98f0cf6af)
@@ -1,12 +0,0 @@
-package com.knight.stream;
-
-public class Test {
-
-  public static void main(String[] args) {
-
-    System.out.println("한글왜깨짐");
-    System.out.println("오류테스트중이야 커밋테스트");
-    System.out.println("테스트중입니다");
-  }
-
-}
